TestFoundation/TestBundle.swift

```diff
@@ -9,6 +9,12 @@
 
 import CoreFoundation
 
+#if DEPLOYMENT_RUNTIME_OBJC || os(Linux) || os(Android)
```

Comment:
We have consolidated these imports elsewhere. Isn't it working?

Reply:
No, unfortunately I need internal symbols that are not available with the `@_exported import Foundation` line in `TestImports.swift`. I also tried there to add `@testable` but it does not work.

Reply:
A change that needs to be made after the last few updates to CI building: you need to guard use of `@testable import` like this:

```swift
#if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT
#if DEPLOYMENT…
@testable import …
// etc
#endif
#endif
```

Tests that make use of internal symbols must also be bracketed by `#if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT`.

Reply:
(Support for this to actually run those tests once bracketed is coming in the next attempt at #1597. Unfortunately, it has been temporarily reverted.)

Reply:
Done! So, after that change is merged we should start a new test for this PR as the current test won't test anything, I suppose 🤔